pipelines/templates/deploy-function-template.yml

```diff
@@ -61,6 +61,7 @@ steps:
           queues = @{
               provisionPosition = "provision-position"
               scheduledNotificationReportQueue = "scheduled-notification"
+              departmentSummaryWeeklyQueue = "department-summary-weekly-queue"
```

Comment:
Shouldn't this be added to a new `deploy-summary-function-template`?

Reply:
Will change when setting up summary function pipelines

Reply:
We fix this in a later task